scripts/check_pre_commit_consistency.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# (C) Copyright Alpine Quantum Technologies GmbH 2024
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# Check consistency between versions of the pre-commit hooks
+# and the packages installed by poetry.
+
+set -euo pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# Retrieve the version of a Python package installed by Poetry.
+installed_package_version() {
+    package_name="$1"
+    # FIXME: install poetry-export plugin to not need to silence the warning here.
+    package_version=$(poetry export \
```

Comment:
Actually I take this back, I think this is just a belt-and-braces habit from me to quote _everything_... There are no instances where this would cause a problem here.